wodles/aws/aws_s3.py

```diff
@@ -2254,6 +2288,14 @@ def reformat_msg(self, event):
             AWSBucket.reformat_msg(self, event)
             yield event
 
+    def load_information_from_file(self, log_key):
+        if log_key.split('.', 1)[1] == 'jsonl.gz':
```

Comment:
The `.` character is safe to use for key names according to the [AWS documentation](https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-keys.html), so this split operation is not recommended.

Reply:
Done